modules/admin_manual/pages/release_notes.adoc

```diff
@@ -27,6 +28,71 @@
+You can also read https://owncloud.org/changelog/server/[the full ownCloud Server changelog] for further details on what has changed.
+
+=== Migrations
+
+Upgrading from ownCloud Server 10.3.x to 10.4.0 does not involve database migrations. The upgrade duration is therefore expected to be short.
+
+=== PHP 7.0 support discontinued
+
+As https://doc.owncloud.com/server/admin_manual/release_notes.html#php-7-0-deprecation-note[announced] with the last releases of ownCloud Server, version 10.4 **does not support PHP 7.0 anymore**. If you're running on PHP 7.0 it is necessary to upgrade PHP **prior** to conducting the upgrade to Server 10.4. It is recommended to upgrade to PHP >= 7.2. See the https://doc.owncloud.com/server/admin_manual/installation/system_requirements.html[system requirements] for more information.
+
+NOTE: If you're using the official Docker containers or the Univention appliance, this has been taken care of already.
+
+=== Expiration date for user and group shares
+
+To give users and administrators more control about access to resources, Server 10.4 introduces an expiration date for user and group shares, just like it's known from Public Links. With this new feature, users can control the lifetime of shares with other users or groups. Administrators can choose to set a default maximum lifetime and to enforce it. To integrate this change, the UI in the user/group sharing tab of the sidebar has been adapted. When a resource is shared, the user and group entries can now be expanded and collapsed using the cog wheel next to the trash bin icon to show/hide the permissions and the expiration date field in order to maintain overview. Additionally, to allow users to recognize expiring shares at a glance, a new clock indicator will be shown next to the cog wheel.
+
+Administrators can configure the feature in the 'Sharing' section of the admin settings.
+
+=== Sharing information in subfolders
+
+ownCloud Server 10.4 puts focus on user awareness for shared areas to prevent accidentally sharing data or changing other users' data as well as to make it easier for users to recognize who has access to shared areas. Practically, users can now better recognize shared resources using a new share overlay indicator on file and folder icons. The indicators are also applied to resources which are not directly shared but are part of a share (when working in a shared folder).
+
+Apart from that, the sharing sidebar panels have been improved to also show users/groups and public links that have access through shares on parent folders. These will be shown as static entries with a "via" indicator that allows users to jump to the parent folder and to change the share properties if desired.
+
+These information are only shown to share owners (users that created shares) as other share recipients are not entitled to get detailed information about who else has access.
```

Comment:
@settermjd what do we do in English:
```suggestion
This information is only shown to share owners (users that created shares) as other share recipients are not entitled to get detailed information about who else has access.
```

But actually this sentence is about the "the pieces of sharing information mentioned above" (plural)

Maybe:
```suggestion
This sharing information is only shown to share owners (users that created shares) as other share recipients are not entitled to get detailed information about who else has access.
```
?